app/src/main/res/values/colors.xml

```diff
@@ -15,11 +15,5 @@
   ~ limitations under the License.
   -->
 <resources>
```

Comment:
Delete this file completely, you will set the status bar color in themes, if you use M3 template.